src/pages/Landing/index.tsx

```diff
@@ -48,34 +80,30 @@ const Glow = styled.div`
   border-radius: 24px;
   max-width: 480px;
   width: 100%;
+  height: 100%;
 `
 
 const ContentContainer = styled.div<{ isDarkMode: boolean }>`
+  position: absolute;
   display: flex;
   flex-direction: column;
   align-items: center;
+  justify-content: end;
   width: 100%;
+  padding: 0 0 40px;
   max-width: min(720px, 90%);
-  position: sticky;
-  bottom: 0;
+  min-height: 500px;
   z-index: ${Z_INDEX.dropdown};
-  padding: 32px 0 80px;
   transition: ${({ theme }) => `${theme.transition.duration.medium} ${theme.transition.timing.ease} opacity`};
-
-  * {
-    pointer-events: auto;
-  }
-
-  @media screen and (min-width: ${BREAKPOINTS.md}px) {
-    padding: 64px 0;
-  }
+  height: ${({ theme }) => `calc(100vh - ${theme.navHeight + theme.mobileBottomBarHeight}px)`};
+  pointer-events: none;
```

Comment:
probably don't need this?

Reply:
this one i do need, because ContentContainer is now layered over the LandingSwap component, so we need to allow hover events to pass through.